### DISCUSSION TITLE
Announce User Points - active tracks

### DISCUSSION THREAD

**Discussion by mariush444** [osmandapp/OsmAnd#13002] [Maps Related]
There is possibility to announce User points like: 
**Track waypoints, Nearby favorities, Nearby POI** in _Voice prompts_ 
**POI , Favorities, Trafic warnings** in _Show along the route_.
during navigation.

Is it possible to add announce and/or show waypoints for all tracks that are activated (shown on the screen)?
It is similar to favorites behavior but when I fallow the one track I cross sometimes other tracks that has important info for me that is close to the cross. 

Currently, I have to edit tracks and move waypoints from one track to another just for one journey/activity.

Thanks in advance

**Comment by vshcherb** [comment]
I think there is an issue for that... Though we plan to have Converter GPX -> POI available and it would be possible to use these waypoints as POI and filter only needed

**Reply by mariush444** [reply]
I'm not sure that POI's works OK.
e.g. https://github.com/osmandapp/OsmAnd/discussions/12252
speed cam is very close to road but there is no announcement because there is no relation to road.

but any solution that will work is welcome :)
